app/models/spree_avatax_certified/address.rb

```diff
@@ -11,132 +10,107 @@ class Address
 
     def initialize(order)
       @order = order
+      @ship_address = order.ship_address
+      @origin_address = JSON.parse(Spree::Config.avatax_origin)
+      @stock_loc_ids = Spree::Stock::Coordinator.new(order).packages.map(&:to_shipment).map(&:stock_location_id)
       @addresses = []
-      @logger ||= AvataxHelper::AvataxLog.new('avalara_order_addresses', 'SpreeAvataxCertified::Address', 'building addresses')
+      @logger ||= AvataxHelper::AvataxLog.new('avalara_order_addresses', 'SpreeAvataxCertified::Address', "Building Addresses for Order#: #{order.number}")
```

Comment:
I think I already commented on that to place it below in a method. It only pollutes the initialize here.